### ISSUE TITLE
Magnifying glass icon is misplaced with .input-search in Ar and He

### ISSUE THREAD

**Issue by MikeTschudi** [Esri/calcite-web#1061]
From our i18n reviewer: "In order to follow the RTL habit, the magnifying glass icon should be on the right side of search box."

En
![image](https://user-images.githubusercontent.com/2125181/52448941-6cf1a380-2aea-11e9-9672-da48355c7c3c.png)

Ar & He
![image](https://user-images.githubusercontent.com/2125181/52448964-7f6bdd00-2aea-11e9-9e99-de0265fc8871.png)

Sample is from documentation; base HTML:
```html
<html>
<head>
  <link rel="stylesheet" href="https://s3-us-west-1.amazonaws.com/patterns.esri.com/files/calcite-web/1.2.4/css/calcite-web.min.css">
</head>
<body>
  <div class="styleguide-modifiers leader-1 trailer-1 margin-left-1 margin-right-1">

    <div class="styleguide-modifier-example">
      <label>
      Label Text
      <input type="text" placeholder="descriptive or sample text" class="input-success" required="">
      </label>
      <code class="modifier-name">.input-success</code>
    </div>

    <div class="styleguide-modifier-example">
      <label>
      Label Text
      <input type="text" placeholder="descriptive or sample text" class="input-error" required="">
      </label>
      <code class="modifier-name">.input-error</code>
    </div>

    <div class="styleguide-modifier-example">
      <label>
      Label Text
      <input type="text" placeholder="descriptive or sample text" class="input-search" required="">
      </label>
      <code class="modifier-name">.input-search</code>
    </div>

    <div class="styleguide-modifier-example">
      <label>
      Label Text
      <input type="text" placeholder="descriptive or sample text" class="input-search input-minimal" required="">
      </label>
      <code class="modifier-name">.input-search input-minimal</code>
    </div>

  </div>
  <script src="https://s3-us-west-1.amazonaws.com/patterns.esri.com/files/calcite-web/1.2.4/js/calcite-web.min.js"></script>
</body>
</html>
```

Ar variant:
```html
<html lang="ar" dir="rtl">
```

He variant:
```html
<html lang="he" dir="rtl">
```

**Comment by MikeTschudi** [comment]
Might this be an acceptable solution?
```html
  html[dir="rtl"] .input-search {
    background-position: right center;
    padding-right: 1.5em;
  }
```

**Comment by paulcpederson** [comment]
@MikeTschudi yeah, looks like we fixed the success/warning icons in #990 but missed the search icons. Can probably add the rtl rule to the base class: https://github.com/Esri/calcite-web/blob/master/lib/sass/calcite-web/components/_form.scss#L1 

I can try to get this onto the next patch release

**Comment by paulcpederson** [comment]
part of 1.2.5
